src/ekore/operator_matrix_elements/polarized/space_like/as2.py

```diff
@@ -0,0 +1,468 @@
+r"""
+This module contains the |NNLO| |OME| in the polarized case for the matching conditions in the |VFNS|.
+
+The equations are given in :cite:`Bierenbaum_2023`.
+As in the |NLO| |OME|, in the paper, an additional factor 2 can be found in front of the anomalous dimensions and factor (-1) for odd powers of L.
+The anomalous dimensions and beta function with the addition 'hat' are defined as in the |NLO| case.
+"""
+import numba as nb
+import numpy as np
+
+from eko.beta import beta_qcd_as2 as beta_0
+from eko.constants import CA, CF, TR
+
+from .... import harmonics
+from ....anomalous_dimensions.polarized.space_like.as1 import gamma_gg as gamma0_gg
+from ....anomalous_dimensions.polarized.space_like.as1 import gamma_gq as gamma0_gq
+from ....anomalous_dimensions.polarized.space_like.as1 import gamma_qg as gamma0_qg
+from ....anomalous_dimensions.unpolarized.space_like.as1 import gamma_ns as gamma0_qq
+from ....harmonics.constants import zeta2, zeta3
+
+# TODO: here we should not call the polygamma explictly
+from ....harmonics.polygamma import cern_polygamma
```

Comment:
@adrianneschauss here we should remove any explicit call to Polygamma. 
This will solve also the (-1)^n business.